src/utils/validators.ts

```diff
@@ -51,12 +52,12 @@ export const getCardNumberError = (cardNumber: string, { errorMessages = {} as a
       return isLuhnValid;
     }
   }
-  return errorMessages.invalidCardNumber || INVALID_CARD_NUMBER;
+  return typeof errorMessages !== 'undefined' ? errorMessages.invalidCardNumber : INVALID_CARD_NUMBER;
```

Comment:
Not sure about this implementation..writing this down, so we don't forget..Thanks.